packages/web/src/app/doc-pages/about/tutorial/angular-tutorial/angular-tutorial.component.html

```diff
@@ -0,0 +1,115 @@
+  </li>
+  <li class="e-mt-24">
+    <div class="e-title-xsmall">Add CSS-reset<span class="e-tag e-ml-8">Optional</span></div>
+    <p class="e-mt-8">
+      Import the Elvis CSS reset if you wish to apply a generic CSS reset to your application.
+    </p>
+    <p class="e-my-8 e-text-md">
+      <strong
+        >Be aware that you may break styles in your app if you import this in an existing application, since
+        you may rely on browser defaults without knowing it. This is why this import is separate from the
+        required Elvis import.</strong
+      >
+    </p>
+    <app-code-viewer [tabs]="['Typescript']" [fileName]="'main.ts'" [activeCode]="codeStep4" hideTabs />
+  </li>
+</ol>
+
+<!--ADVANCED TUTORIAL-->
+<h3 class="e-title-small e-mt-64">Add an advanced component</h3>
+<ol class="e-list e-list--numbers e-mt-40">
+  <li>
+    <div class="e-title-xsmall">Install component</div>
+    <p class="e-mt-8">
+      Install the
+      <a [routerLink]="['/components/datepicker']" class="e-link e-link--inline">datepicker component.</a>
+    </p>
+    <p class="e-my-8 e-text-md">
+      <strong
+        >The packages acts like any other external dependency you install and update in your project.
+      </strong>
+    </p>
+    <app-code-viewer [tabs]="['Bash']" [fileName]="'Terminal'" [activeCode]="codeStepAdvanced1" hideTabs />
+  </li>
+  <li class="e-mt-24">
+    <div class="e-title-xsmall">Add imports</div>
+    <p class="e-my-8">Import the component.</p>
+    <app-code-viewer
+      [tabs]="['Typescript']"
+      [fileName]="'main.ts'"
+      [activeCode]="codeStepAdvanced2"
+      hideTabs
+    />
+  </li>
+  <li class="e-mt-24">
+    <div class="e-title-xsmall">Add schemas</div>
+    <p class="e-my-8">Import the custom-element schema to make the web components work.</p>
+    <app-code-viewer
+      [tabs]="['Typescript']"
+      [fileName]="'app.module.ts / app.component.ts (standalone)'"
+      [activeCode]="codeStepAdvanced3"
+      hideTabs
+    />
+  </li>
+  <li class="e-mt-24">
+    <div class="e-title-xsmall">Start using advanced components</div>
+    <p class="e-my-8">
+      Choose between several different advanced components and customize according to your needs by using the
+      available props.
+    </p>
+    <app-code-viewer
+      [tabs]="['HTML']"
+      [fileName]="'app.component.html'"
+      [activeCode]="codeStepAdvanced4"
+      hideTabs
+    />
+  </li>
+  <li class="e-mt-24">
+    <div class="e-title-xsmall">Using props and events</div>
+    <p class="e-my-8">
+      Try adding the fullWidth prop to see the datepicker take up full width of available space. Try adding
+      valueOnChange event to see the events coming in when the datepicker changes. The
+      <a [routerLink]="['/components/datepicker/']" fragment="API" class="e-link e-link--inline"
+        >datepicker API</a
+      >
+      lists all the properties and events available for datepicker.
```

Comment:
Should the "code" words be wrapped in a `<code>`-element?

```suggestion
      Try adding the <code>fullWidth</code>-prop to see the datepicker take up the full width of available space. Try adding
      <code>valueOnChange</code>-event to see the events coming in when the datepicker changes. The
      <a [routerLink]="['/components/datepicker/']" fragment="API" class="e-link e-link--inline"
        >datepicker API</a
      >
      lists all the properties and events available for datepicker.
```